src/qdomyos-zwift.pro

```diff
@@ -9,7 +9,11 @@ qtHaveModule(httpserver) {
     HEADERS += webserverinfosender.h
 }
 
-CONFIG += c++11 console debug app_bundle
+CONFIG += c++17 console app_bundle optimize_full ltcg
+unix:android: {
```

Comment:
@lifof why only on android?

Reply:
@cagnulein 
if you try to compile the android version including the optimize_full and ltcg while excluding the debug in CONFIG you'll get an error at link time that says something like : 
> link time optimization only supports -O0 -> -O3

and after I investigated this matter I found that in the official Qt 5.15.2 that I have on my Linux machine :
> mkspecs/android-clang/qmake.conf

on the lines 58 and 59 the optimization flags that are used are always -Oz, so that's why only on android I removed the -Oz to add -O3

`QMAKE_CFLAGS_OPTIMIZE      = -Oz`
`QMAKE_CFLAGS_OPTIMIZE_FULL = -Oz`

Hhhh that's too long of an explanation

Reply:
ok thanks!